### PR TITLE
reword all rules

### PR DESCRIPTION
**Pull request by mlutze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,25 +39,25 @@ they may be played on any card and any card may be played on them.
...
+they may instead play a collection of cards such that:
 - the first card alone would be valid to play
 - the remaining cards are in increasing order
 - all the cards have prime values, where a Jack, Queen, King, and Ace have the values 11, 12, 13, and 14, respectively.
 
 ## Give 2
 _Added by Jonathan, 14 Aug 2023_
 
-When you play a 2, you must designate two other players:
+When player plays a 2, they must designate two other players:
```
Comment (JonathanStarup) [inline README.md]: a player

File: README.md [hunk 2]
```diff
@@ -62,34 +66,34 @@ the Giver selects two cards from their hand and gives them to the Receiver.
...
-If you begin to have two cards, then you must say "Toktok" before someone next plays a card.
+If a player begins to have two cards, then they must say "Toktok" before a player next plays a card.
 
 ## Delay of Game
 _Added by Matt, 21 Aug 2023_
 
-You may at any point raise your hand to indicate that the current player is taking too long.
-If the majority of players raise their hands, the current player draws a penalty card.
+A player may at any point raise their hand to indicate that the current player is taking too long.
+If the majority of players raise their hands, the Pending Player draws a penalty card.
```
Comment (JonathanStarup) [inline README.md:90]: Pending or active player

File: README.md [hunk 3]
```diff
@@ -109,80 +113,80 @@ you may instead play a collection of cards such that:
...
 
-When you play a trap card, you may say a card value one step higher or lower than the card's value.
+When a player plays a trap card, they may say a card value one step higher or lower than the card's value.
 The card is then considered to have that value.
 
 ## 6 Splits
 _Added by Matt, 20 Sep 2023_
 
-When you play a 6, you must play the card in a new pile.
+When a player plasy a 6, they must play the card in a new pile.
```
Comment (JonathanStarup) [inline README.md]: plays*

**Review by JonathanStarup — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (mlutze) [reply README.md:90]: Pending. We can be waiting for someone to do something even if it's not their turn.

**Pull request merged**